src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import getAllArticles from "../api";
+
+const ArticlesList = () => {
+    const [articlesList, setArticlesList] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        getAllArticles().then((allArticles) => {
+            setArticlesList(allArticles)
+            setIsLoading(false)
+        })
+    }, [])
+
+    if (isLoading) {
+        return <h2>Loading...</h2>
+    }
+    else {
+        return (
+          <main>
+                <h2>Articles</h2>
+            <ul>
+              {articlesList.map((article) => {
+                  return (
+                    <li key={article.article_id} className="articles">
```

Comment:
You have two different components where you make this li element - a sort of ArticleCard
Using something in multiple places is usually a good sign it can be turned into it's own component - I recommend doing this but it's not mandatory 🔍